Create members list

JamesVorder opened the pull request:
How does this look to you two?

Commit:
Create members list

stephen304 reviewed (approved):
lgtm. Would be good to also add push restrictions in the settings

JamesVorder commented:
I added push restrictions, and added us all to the "Core" team. We alone have write access to this repo, but it is publicly visible. The push restrictions are a little crude... It is just restricted to getting two approvals before merge, not specifically linked to _who_ should do the approving. You're both admins, maybe you know a way to do that?

cure reviewed (approved):
LGTM thanks. I think you've done as much as possible with the push restrictions, they are not very sophisticated.
